### PR TITLE
Added Simplified Chinese support

### PR DESCRIPTION
**Pull request by seanzhang98**

### CODE REVIEW

**Comment by PunkyBust** [comment]
Hello,
Same here but for french, I don't find this file on my deck.
And have no hand on github side, so how can we add our localization please ? fr-FR needed on my side.
Thanks  :)

**Comment by TrainDoctor** [comment]
We are currently setting up a weblate to help with translation @PunkyBust. We will file an issue and pin it for future people who want to contribute translations.

**Comment by PunkyBust** [comment]
OK great, understand better now. Happy to contribute if I can help for french translation

File: backend/locales/zh-CN.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      "testing": "测试"
+    }
+  },
+  "Updater": {
+    "decky_updates": "Decky 更新",
+    "no_patch_notes_desc": "此版本没有补丁说明",
+    "patch_notes_desc": "补丁说明",
+    "updates": {
+      "check_button": "检查更新",
+      "checking": "'检查中",
```
Comment (RodoMa92) [inline backend/locales/zh-CN.json:174]: The initial ' after " was a typo in the english translation, please remove that.

**Review by RodoMa92 — CHANGES_REQUESTED** [review]
See my comment below.

**Comment by seanzhang98** [comment]
Updated the config file and the translation as well.

**Comment by RodoMa92** [comment]
Just to be sure that there is no translation strings missing for the framework, can you please try to run the activity extract text (Control + Shift + P, insert activity enter, insert extracttext enter) and eventually complete the new strings added to the json file? After that this should be good to merge to me.

**Comment by seanzhang98** [comment]
Not sure how to run the activity extract text on my set up. Tried Control + Shift + P but nothing happened.

**Comment by RodoMa92** [comment]
Here you can find a guide on tasks (my translation in my language is different, sorry) https://code.visualstudio.com/docs/editor/tasks

**Comment by RodoMa92** [comment]
Just tested personally, it's fine as keys goes. It can go in now.

**Comment by RodoMa92** [comment]
I've manually added your translation to the weblate instance that the team brought up recently, it should be in shortly in the loader itself. You can find it here if you want to add/correct any mistake made by my copy and paste :P : https://weblate.werwolv.net/projects/decky/decky/

**Pull request closed without merging**